llvm/test/Feature/Repo/Inputs/remove_GO_ticketnode.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import re
+
+def filter_definition (line, GOList, filterStr):
+  if any(f in line for f in GOList):
+    return re.sub(filterStr, '', line)
+  return line
+
+def remove_GO_ticketnode (GOList, isFunction, inputFile, outputFile):
+  """
+  Remove the GO's repo_ticket metadata.
+  """
+  filterStr = "!repo_ticket ![0-9]+"
+  if isFunction:
+    GOList = ["@{0}()".format(i) for i in GOList]
+  else:
+    GOList = ["@{0} =".format(i) for i in GOList]
+    filterStr = ", " + filterStr
```

Comment:
Why do variables get a leading comma?

Reply:
In the IR code, a function definition is (no comma):
```
define i32 @A() !repo_ticket !0 {
```

A variable definition is:
```
@str = private constant [6 x i8] c"Ipsum\00", align 1, !repo_ticket !0
``` 
We need to remove `, !repo_ticket !0`. If not removed the comma,  the IR code can't be compiled.

Reply:
It’s a regular expression, so could it be ",? !repo_ticket ![0-9]+" (i.e. make a leading comma optional)?

Reply:
Good point, thanks. Changed. 
After changing this, I can combine the `-f` and `-v` python arguments into one argument `-r`, which could remove the global object.